tests/ixia/test_ixia_traffic_restpy.py

```diff
@@ -0,0 +1,153 @@
+###############################################################################
+# This test cases demonstrates: 
+#   * All the fixtures required for running ixia script (please see the 
+#     arguments of the test function)
+#   * How Ixia chassis card/ports are addressed
+#   * How you can configure/control ixia devices, start traffic and collect 
+#     statistics using REST API
+#   * This simple sanity test cases can be used to check if testbed setup
+#     is correct or not - since it prints a lot of testbed data
+###############################################################################
+
+import logging
+import time
+import pytest
+import ipaddr
+from common.utilities import wait_until
+from common.fixtures.conn_graph_facts import conn_graph_facts, fanout_graph_facts 
+from common.reboot import *
+from ixnetwork_restpy import SessionAssistant, Files
+
+from common.ixia.ixia_fixtures import ixia_api_serv_ip, ixia_api_serv_user,\
+     ixia_api_serv_passwd, ixia_api_serv_port, ixia_api_serv_session_id, \
+     ixia_api_server_session  
+
+from common.ixia.ixia_helpers import get_neigh_ixia_mgmt_ip, get_neigh_ixia_card,\
+     get_neigh_ixia_port, IxiaFanoutManager
+
+import time
+
+def create_ipv4_traffic_end_points (
```

Comment:
This should get moved to the common lib

Reply:
I have changed and moved the function in the library